File: Examples/CSharp/Programming-Documents/Ranges/RangesGetText.cs
using System;
using System.Collections;
using System.IO;
using Aspose.Words;
using Aspose.Words.Tables;
using Aspose.Words.Fields;
using Aspose.Words.Layout;

namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Ranges
{
    class RangesGetText
    {
        public static void Run()
        {
            //ExStart:RangesGetText
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithRanges();

            Document doc = new Document(dataDir + "Document.doc");
            string text = doc.Range.Text; 
            //ExEnd:RangesGetText
            Console.WriteLine("\nDocument have following text range " + text);
        }
    }
}

File: Examples/VisualBasic/VisualBasic/Programming-Documents/Ranges/RangesGetText.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Words
Public Class RangesGetText
    Public Shared Sub Run()
        ' ExStart:RangesGetText
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithRanges()

        Dim doc As New Document(dataDir & Convert.ToString("Document.doc"))
        Dim text As String = doc.Range.Text
        ' ExEnd:RangesGetText
        Console.WriteLine(Convert.ToString(vbLf & "Document have following text range ") & text)
    End Sub
End Class
